src/Components/Person/Person.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './Person.scss';
+import PropTypes from 'prop-types';
+
+export function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) {
+  const partnerType = sex === 'm'
+    ? 'wife'
+    : 'husband';
+  const partner = isMarried
+    ? `My ${partnerType} name is ${partnerName}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+      <p className="Person__partner">{partner}</p>
+    </section>
+  );
+}
+
+Person.defaultProps = {
+  age: null,
+  partnerName: null,
```

Comment:
but why?

Reply:
If I am putting default age as 0, it is showing 0 [https://imgur.com/dAGHz5s](url)